docs/semgrep-supply-chain/dependency-search.md

```diff
@@ -72,6 +72,33 @@ Dependency search provides the following filters, which correspond to the data p
 ![Screenshot of dependency search with query](/img/SSC-DepSearch-Query.png#md-width)
 _**Figure**. Dependency search page with sample search query._
 
+## Dependency graphs (beta)
+
+:::info
+Semgrep does not support the generation of dependency graphs for managed scans of projects.
```

Comment:
```suggestion
Semgrep does not support the generation of dependency graphs for managed scans of projects.
```
Are we referring to Semgrep Managed Scans here? Just checking.

Reply:
Correct